client/spec/components/Avatar.spec.jsx

```diff
@@ -1,46 +1,38 @@
-import avatarFactory from "inject!Avatar";
-import ReactDOM from "react-dom";
+import Avatar from "Avatar";
 
 describe( "Avatar", () => {
-	let component, dependencies;
+	let component, container;
 
 	beforeEach( () => {
-		dependencies = {};
-
-		const Avatar = avatarFactory( dependencies );
-
-		component = ReactUtils.renderIntoDocument( <Avatar /> );
+		container = document.createElement( "div" );
 	} );
 
-	afterEach( () => {
-		if ( component ) {
-			ReactDOM.unmountComponentAtNode( ReactDOM.findDOMNode( component ).parentNode );
-		}
-	} );
+	function render( props ) {
+		ReactDOM.render( <Avatar {...props} />, container );
```

Comment:
Render returns `null`. I am pretty sure its because its returning a DOM node, and its a stateless component.